perftracker/views.py

```diff
@@ -158,6 +156,48 @@ def pt_comparison_tables_info_json(request, api_ver, project_id, cmp_id, group_i
     return JsonResponse(ret, safe=False)
 
 
+@csrf_exempt
+def pt_comparison_section_properties_save(request, api_ver, project_id, cmp_id, group_id, section_id):
+    try:
+        data = json.loads(request.body.decode("utf-8"))
+    except json.JSONDecodeError:
+        return HttpResponseBadRequest("Wrong json")
+
+    data = data[str(section_id)]
+    PTTrainDataModel.pt_validate_json(data)
+
+    try:
+        formatted_data = '|'.join([f'{x};{y}' for x, y in data['data']])
```

Comment:
I suspect there are possibilities for x to be non-number. 
Probably you need to check for that and don't analyze such sections.